File: PHP/Events/AvailableTicketInfo.php
<?php
namespace LTD_API;

class AvailableTicketInfo
{
    private $venueId;
    private $venueName;
    private $venueAddress;
    private $venueCity;
    private $ticketName1;
    private $ticketName2;
    private $adjacencyIndex;
    private $faceValue;
    private $sellingPrice;
    private $directBookingUrl;
    private $ticketArea;
    private $ticketId;
    private $isRestrictedView;
    private $restrictionDescription;
    
    public function getVenueId()
    {
        return $this->venueId;
    }
    public function setVenueId($venueId)
    {
        $this->venueId = $venueId;
    }

    public function getVenueName()
    {
        return $this->venueName;
    }
    public function setVenueName($venueName)
    {
        $this->venueName = $venueName;
    }

    public function getVenueAddress()
    {
        return $this->venueAddress;
    }
    public function setVenueAddress($venueAddress)
    {
        $this->venueAddress = $venueAddress;
    }

    public function getVenueCity()
    {
        return $this->venueCity;
    }
    public function setVenueCity($venueCity)
    {
        $this->venueCity = $venueCity;
    }

    public function getTicketName1()
    {
        return $this->ticketName1;
    }
    public function setTicketName1($ticketName1)
    {
        $this->ticketName1 = $ticketName1;
    }

    public function getTicketName2()
    {
        return $this->ticketName2;
    }
    public function setTicketName2($ticketName2)
    {
        $this->ticketName2 = $ticketName2;
    }

    public function getAdjacencyIndex()
    {
        return $this->adjacencyIndex;
    }
    public function setAdjacencyIndex($adjacencyIndex)
    {
        $this->adjacencyIndex = $adjacencyIndex;
    }

    public function getFaceValue()
    {
        return $this->faceValue;
    }
    public function setFaceValue($faceValue)
    {
        $this->faceValue = $faceValue;
    }

    public function getSellingPrice()
    {
        return $this->sellingPrice;
    }
    public function setSellingPrice($sellingPrice)
    {
        $this->sellingPrice = $sellingPrice;
    }

    public function getDirectBookingUrl()
    {
        return $this->directBookingUrl;
    }
    public function setDirectBookingUrl($directBookingUrl)
    {
        $this->directBookingUrl = $directBookingUrl;
    }

    public function getTicketArea()
    {
        return $this->ticketArea;
    }
    public function setTicketArea($ticketArea)
    {
        $this->ticketArea = $ticketArea;
    }

    public function getTicketId()
    {
        return $this->ticketId;
    }
    public function setTicketId($ticketId)
    {
        $this->ticketId = $ticketId;
    }

    public function getIsRestrictedView()
    {
        return $this->isRestrictedView;
    }
    public function setIsRestrictedView($isRestrictedView)
    {
        $this->isRestrictedView = $isRestrictedView;
    }

    public function getRestrictionDescription()
    {
        return $this->restrictionDescription;
    }
    public function setRestrictionDescription($restrictionDescription)
    {
        $this->restrictionDescription = $restrictionDescription;
    }
}

File: Java/Events/AvailableTicketInfo.java
package LTD_API;

public class AvailableTicketInfo
{
    private int venueId;
    private String venueName;
    private String venueAddress;
    private String venueCity;
    private String ticketName1;
    private String ticketName2;
    private int adjacencyIndex;
    private decimal faceValue;
    private decimal sellingPrice;
    private String directBookingUrl;
    private String ticketArea;
    private int ticketId;
    private bool isRestrictedView;
    private String restrictionDescription;
    
    public int getVenueId()
    {
        return venueId;
    }
    public void setVenueId(int venueId)
    {
        this.venueId = venueId;
    }

    public String getVenueName()
    {
        return venueName;
    }
    public void setVenueName(String venueName)
    {
        this.venueName = venueName;
    }

    public String getVenueAddress()
    {
        return venueAddress;
    }
    public void setVenueAddress(String venueAddress)
    {
        this.venueAddress = venueAddress;
    }

    public String getVenueCity()
    {
        return venueCity;
    }
    public void setVenueCity(String venueCity)
    {
        this.venueCity = venueCity;
    }

    public String getTicketName1()
    {
        return ticketName1;
    }
    public void setTicketName1(String ticketName1)
    {
        this.ticketName1 = ticketName1;
    }

    public String getTicketName2()
    {
        return ticketName2;
    }
    public void setTicketName2(String ticketName2)
    {
        this.ticketName2 = ticketName2;
    }

    public int getAdjacencyIndex()
    {
        return adjacencyIndex;
    }
    public void setAdjacencyIndex(int adjacencyIndex)
    {
        this.adjacencyIndex = adjacencyIndex;
    }

    public decimal getFaceValue()
    {
        return faceValue;
    }
    public void setFaceValue(decimal faceValue)
    {
        this.faceValue = faceValue;
    }

    public decimal getSellingPrice()
    {
        return sellingPrice;
    }
    public void setSellingPrice(decimal sellingPrice)
    {
        this.sellingPrice = sellingPrice;
    }

    public String getDirectBookingUrl()
    {
        return directBookingUrl;
    }
    public void setDirectBookingUrl(String directBookingUrl)
    {
        this.directBookingUrl = directBookingUrl;
    }

    public String getTicketArea()
    {
        return ticketArea;
    }
    public void setTicketArea(String ticketArea)
    {
        this.ticketArea = ticketArea;
    }

    public int getTicketId()
    {
        return ticketId;
    }
    public void setTicketId(int ticketId)
    {
        this.ticketId = ticketId;
    }

    public bool getIsRestrictedView()
    {
        return isRestrictedView;
    }
    public void setIsRestrictedView(bool isRestrictedView)
    {
        this.isRestrictedView = isRestrictedView;
    }

    public String getRestrictionDescription()
    {
        return restrictionDescription;
    }
    public void setRestrictionDescription(String restrictionDescription)
    {
        this.restrictionDescription = restrictionDescription;
    }
}
